docker-compose.yml

```diff
@@ -0,0 +1,170 @@
+version: '2.4'
+services:
+
+  router:
+    image: "${ROUTER_DOCKER_IMAGE}"
+    ports:
+      - 80:80
+      - 443:443
+    restart: unless-stopped
+    volumes:
+      - "${CARTO3_ONPREM_VOLUMES_BASE_PATH}certs/:/etc/nginx/ssl/:ro"
+    env_file:
+      - .env
+    logging:
+      driver: "local"
+      options:
+        max-size: "5M"
+        max-file: "20"
+
+  accounts-www:
+    image: "${ACCOUNTS_WWW_DOCKER_IMAGE}"
+    env_file:
+      - .env
+    restart: unless-stopped
+    logging:
+      driver: "local"
+      options:
+        max-size: "5M"
+        max-file: "20"
+
+  workspace-www:
+    image: "${WORKSPACE_WWW_DOCKER_IMAGE}"
+    env_file:
+      - .env
+    restart: unless-stopped
+    logging:
+      driver: "local"
+      options:
+        max-size: "5M"
+        max-file: "20"
+
+  workspace-api:
+    image: "${WORKSPACE_API_DOCKER_IMAGE}"
+    env_file:
+      - .env
+    depends_on:
+      - workspace-postgres
+      - workspace-migrations
+      - redis
+    ports:
+      - 8001:3000
+    command: npm run start:built
+    volumes: &override-sa
+      - "${CARTO3_ONPREM_VOLUMES_BASE_PATH}certs/:/usr/src/certs/:ro"
```

Comment:
I think this volume should be mounted in `/usr/src/app/certs/` inside the container, otherwise it fails

Reply:
it depends on `GOOGLE_APPLICATION_CREDENTIALS=../certs/key.json` which has the `..` to jump to the parent dir. It was like that in the dev-env

Reply:
You are in rigth